cdap-ui/graphql/helpers/BatchEndpoints/nextRuntime.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { constructUrl } from 'server/url-helper';
+import { getCDAPConfig } from 'server/cdap-config';
+import { getPOSTRequestOptions, requestPromiseWrapper } from 'gql/resolvers-common';
+
+let cdapConfig;
+getCDAPConfig().then(function(value) {
+  cdapConfig = value;
+});
+
+export async function batchNextRuntime(req, auth) {
+  const namespace = req[0].namespace;
+  const options = getPOSTRequestOptions();
+  options.url = constructUrl(cdapConfig, `/v3/namespaces/${namespace}/nextruntime`);
+  const body = req.slice(0, 25).map((reqObj) => reqObj.program);
+  options.body = body;
+
+  let nextRuntimeInfo = await requestPromiseWrapper(options, auth);
+
+  const runsMap = {};
+  nextRuntimeInfo.forEach((run) => {
+    if (run.schedules) {
+      runsMap[run.appId] = run.schedules;
+    } else {
+      runsMap[run.appId] = [];
```

Comment:
should this be empty object? I see in the schema, the response structure per schedule is 
```
{
  id,
  time
}
```

Reply:
the nextruntime is an array as defined [here](https://github.com/cdapio/cdap/pull/11912/files#diff-0bab0b762798a2c7bda6f7cea7cc72c2R28)